serializer.rst

```diff
@@ -41,6 +41,18 @@ you need it or it can be used in a controller::
         }
     }
 
+.. versionadded:: 5.3
```

Comment:
I wasn't sure if this would be best here in the usage block or if it should have its own section. I'll squash the commits after some feedback